hierarchicalforecast/utils.py

```diff
@@ -190,14 +194,40 @@ def aggregate(
             "Don't set this argument to suppress this warning.",
             category=DeprecationWarning,
         )
+    
+    # Define acceptable aggregation functions
+    acceptable_aggregations = {
+        'sum', 'mean', 'median', 'min', 'max', 'count', 'std', 'var', 'first', 'last'
+    }    
```

Comment:
I do not think this is needed—we can just let pandas raise an `AttributeError` when aggregating rather than raising our own `ValueError`.

Plus, this gives us the flexibility to use custom (anonymous) functions rather than just string function names.